config/src/config/api_config.rs

```diff
@@ -77,6 +81,7 @@ impl Default for ApiConfig {
             max_events_page_size: DEFAULT_MAX_PAGE_SIZE,
             max_account_resources_page_size: DEFAULT_MAX_ACCOUNT_RESOURCES_PAGE_SIZE,
             max_account_modules_page_size: DEFAULT_MAX_ACCOUNT_MODULES_PAGE_SIZE,
+            max_gas_view_function: DEFAULT_MAX_VIEW_GAS,
```

Comment:
this needs to be a configurable parameter before we can ship this

Reply:
I assume that this is configurable as it's in the config section?

Reply:
Yes, this is configurable.